app/javascript/controllers/hello_controller.js

```diff
@@ -1,7 +1,50 @@
-import { Controller } from "@hotwired/stimulus"
+import {Controller} from "@hotwired/stimulus"
 
 export default class extends Controller {
-  connect() {
-    this.element.textContent = "Hello World!"
-  }
+    static targets = ["form"]
+    static values = {url: Array, counter: Number, type: String}
+
+    connect() {
+        this.formList = [];
+        $('input[name="daterange"]').daterangepicker(
+            {
+                locale: {
+                    cancelLabel: 'Clear',
+                    format: 'YYYY-MM-DD'
+                },
+                startDate: '2022-10-01',
+                endDate: '2022-10-26'
+            },
+            function (start, end, label) {
+                alert("A new date range was chosen: " + start.format('YYYY-MM-DD') + ' to ' + end.format('YYYY-MM-DD'));
+            });
+    }
```

Comment:
you have not modified it still using Jquery here also indentation issue

Reply:
removed